app/src/main/java/net/sourceforge/opencamera/ui/DrawPreview.java

```diff
@@ -1240,7 +1240,7 @@ else if( time_ms - ae_started_scanning_ms > 500 ) {
         }
 
         // RecSync leader-client information
-        if( applicationInterface instanceof ExtendedAppInterface ) {
+        if( ((ExtendedAppInterface) applicationInterface).isSoftwareSyncRunning() ) {
```

Comment:
I would actually prefer to set the `applicationInterface` field to store `ExtendedAppInterface`, but I failed to do so, because this class is initialized inside `MyApplicationInterface`. I'll bring the check back for now.